src/views/KeyboardShortcuts.vue

```diff
@@ -64,11 +64,15 @@ export default {
 <style lang="scss" scoped>
 ::v-deep .modal-wrapper .modal-container {
 	display: block;
-	overflow: hidden;
+	overflow: scroll;
 	transition: transform 300ms ease;
 	border-radius: var(--border-radius-large);
 	box-shadow: 0 0 40px rgba(0,0,0,0.2);
 	padding: 30px 40px 20px;
 }
+dt {
```

Comment:
@skjnldsv I remember something about slower performance if the selector is an element rather than an ID or class. Is this the case?

Reply:
It was something about id vs classes. But shouldn't be that significant for such low amount of doms